packages/typography/src/Label.tsx

```diff
@@ -18,7 +18,7 @@ const labelTypeScale: Record<BaseFontSize, string> = {
   `,
   [BaseFontSize.Body2]: css`
     font-size: ${typeScales.body2.fontSize}px;
-    line-height: ${typeScales.body2.lineHeight}px;
+    line-height: 20px;
```

Comment:
Why is this 20px? Might be worth chatting to Rob about

Reply:
I'll discuss with him during the design review